src/schema.json

```diff
@@ -0,0 +1,96 @@
+{
+	"definitions": {
+		"mode": {
+			"type": "string",
+			"enum": [
+				"dev",
+				"dist",
+				"test"
```

Comment:
Does it actually support building a test bundle?

Reply:
I believe so when building CE: https://github.com/dojo/cli-build-widget#test-mode